README.md

```diff
@@ -46,13 +45,26 @@ Learn more about libp2p at [**libp2p.io**](https://libp2p.io) and follow libp2p'
 ```
 nimble install libp2p
 ```
+You'll find the nim-libp2p documentation [here](https://vacp2p.github.io/nim-libp2p/docs/). See [examples](./examples) for simple usage patterns.
 
 ## Getting Started
-You'll find the nim-libp2p documentation [here](https://vacp2p.github.io/nim-libp2p/docs/). See [examples](./examples) for simple usage patterns.
+Try out the chat example. For this you'll need to have [`go-libp2p-daemon`](examples/go-daemon/daemonapi.md) running. Full code can be found [here](https://github.com/status-im/nim-libp2p/blob/master/examples/chat.nim):
```

Comment:
This point is interesting! I'm thinking that later we should work to make the examples not require the daemonapi. I find it somewhat strange that we require a third-party software to be able to run the chat example